### PR TITLE
chore(rxjs): update rxjs to 6.3.2

### PR DESCRIPTION
**Pull request by benlesh**
Updates CLI to 6.3.2

NOTE: There were two typings that needed to be updated here. One was an honest fix due to improved typings and the limitations of TypeScript, the other is a bit more curious (See the changes to base.ts). There's apparently a problem with spreading an array of operators into `pipe`, but only in this case. I'm working on replicating the issue outside of this project, but I haven't been able to do so.

### CODE REVIEW

File: packages/angular_devkit/core/src/json/schema/visitor.ts [hunk 1]
```diff
@@ -69,7 +69,7 @@ function _visitJsonRecursive<ContextT>(
       ? value as Observable<JsonValue>
       : observableOf(value as JsonValue)
   ).pipe(
-    concatMap((value: JsonValue) => {
+    concatMap((value): Observable<JsonValue> => {
```
Comment (hansl) [inline packages/angular_devkit/core/src/json/schema/visitor.ts]: That’s not valid TypeScript is it?

Comment (hansl) [reply packages/angular_devkit/core/src/json/schema/visitor.ts]: It's missing the input type, or doesn't it need it?

Comment (benlesh) [reply packages/angular_devkit/core/src/json/schema/visitor.ts]: It's what Rado recommended in these situations. Unfortunately, TS can't infer the return in these cases where there's branching code.

Comment (benlesh) [reply packages/angular_devkit/core/src/json/schema/visitor.ts]: The input type is inferred.

**Comment by alan-agius4** [comment]
@benlesh Thanks for this. However the E2E's will fail due to an issue that we had a chat about some time ago

https://github.com/ReactiveX/rxjs/issues/4135

File: packages/schematics/schematics/package.json [hunk 2]
```diff
@@ -11,6 +11,7 @@
   "schematics": "./collection.json",
   "dependencies": {
     "@angular-devkit/core": "0.0.0",
-    "@angular-devkit/schematics": "0.0.0"
+    "@angular-devkit/schematics": "0.0.0",
+    "rxjs": "6.3.2"
```
Comment (alan-agius4) [inline packages/schematics/schematics/package.json]: Why does this need to be added here? AFAIK schematics/schematics do not directly depend on `rxjs`

File: packages/angular_devkit/schematics/package.json [hunk 3]
```diff
@@ -14,6 +14,6 @@
   ],
   "dependencies": {
     "@angular-devkit/core": "0.0.0",
-    "rxjs": "6.2.2"
+    "rxjs": "6.3"
```
Comment (alan-agius4) [inline packages/angular_devkit/schematics/package.json]: Shouldn't this be `6.3.2` for consistence? As this will be resolved to `6.3.x`

**Comment by benlesh** [comment]
Closing in favor of https://github.com/angular/angular-cli/pull/12362

**Pull request closed without merging**